<launch>
    <arg name="sim" default="false"/>

    <let if="$(env IS_ROBOT false)" name="taskset" value="taskset -c 5"/>
    <let unless="$(env IS_ROBOT false)" name="taskset" value=""/>

    <group unless="$(var sim)">
        <node name="walk_support_state_detector" pkg="bitbots_odometry" exec="walk_support_state_detector" launch-prefix="$(var taskset)">
        <!-- for the normal walk_support_state_detector_default values are set and do not need to be loaded-->
        </node>
    </group>
    <group if="$(var sim)">
        <node name="walk_support_state_detector" pkg="bitbots_odometry" exec="walk_support_state_detector" launch-prefix="$(var taskset)">
            <param name="use_sim_time" value="$(var sim)"/>
            <param from="$(find-pkg-share bitbots_odometry)/config/odometry_config_sim.yaml"/>
        </node>
    </group>
</launch>
